pkg/test/resourcefixture/testdata/basic/metastore/v1alpha1/metastoreservice/metastoreservice-minimal/_http.log

```diff
@@ -0,0 +1,225 @@
+GET https://metastore.googleapis.com/v1/projects/${projectId}/locations/us-central1/services/metastoreservice-minimal-${uniqueId}?%24alt=json%3Benum-encoding%3Dint
+Content-Type: application/json
+User-Agent: kcc/${kccVersion} (+https://github.com/GoogleCloudPlatform/k8s-config-connector) kcc/controller-manager/${kccVersion}
+X-Goog-Request-Params: name=projects%2F${projectId}%2Flocations%2Fus-central1%2Fservices%2Fmetastoreservice-minimal-${uniqueId}
+
+404 Not Found
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "error": {
+    "code": 404,
+    "errors": [
+      {
+        "domain": "global",
+        "message": "Resource 'projects/${projectId}/locations/us-central1/services/metastoreservice-minimal-${uniqueId}' was not found",
+        "reason": "notFound"
+      }
+    ],
+    "message": "Resource 'projects/${projectId}/locations/us-central1/services/metastoreservice-minimal-${uniqueId}' was not found",
+    "status": "NOT_FOUND"
+  }
+}
+
+---
+
+POST https://metastore.googleapis.com/v1/projects/${projectId}/locations/us-central1/services?%24alt=json%3Benum-encoding%3Dint&serviceId=metastoreservice-minimal-${uniqueId}
```

Comment:
I don't see a `PATCH` request after the `POST`.

Reply:
reran with a proper update.yaml. Has patch now.